packages/components/src/components/hds/app-footer/legal-links.ts

```diff
@@ -5,7 +5,23 @@
 
 import Component from '@glimmer/component';
 
-export default class HdsAppFooterLegalLinksComponent extends Component {
+export interface HdsAppFooterLegalLinksSignature {
+  Args: {
+    ariaLabel?: string;
+    hrefForTerms?: string;
+    hrefForPrivacy?: string;
+    hrefForSecurity?: string;
+    hrefForSupport?: string;
+    hrefForAccessibility?: string;
+    year?: string;
+  };
+  Blocks: {
+    default: [];
+  };
+  Element: HTMLUListElement;
```

Comment:
Probably more "correct" this way?
```suggestion
  Element: HdsAppFooterItemSignature['Element'];
```
(if you accept the suggestion, don't forget to import `HdsAppFooterItemSignature`)

Reply:
From the Docs I think what I had was correct?

> The Element field declares what type of element(s), if any, the component applies its passed ...attributes to. This is often the component's root element.

Reply:
@alex-ju I thought that, since the "root" component in the template is a `<Hds::AppFooter::Item>` then the `Element` in the signature would be ok to be `HdsAppFooterItemSignature['Element']`, but apparently not. What am I missing?

Reply:
According to the docs:
> If no Element is specified, it will be a type error to set any HTML attributes or modifiers when invoking your component.
So yes, it's possible to leave out the `Element` entry.  I am going to resolve this, then. @Dhaulagiri ignore my suggestion above.

Reply:
Ended up leaving this as `HTMLUListElement`